src/components/output/Output.vue

```diff
@@ -4,12 +4,15 @@
       <component :is="outputComponent" v-if="outputComponent" :key="presentation.id" :clear="$store.isClear" :alpha="alpha" :presentation="presentation" />
     </Transition>
   </div>
+  <messageOutput v-if="id == 'beamer'" />
```

Comment:
```suggestion

  <messageOutput v-if="id === 'beamer'" />
```

Altijd `===` gebruiken i.p.v. `==`

Daarnaast denk ik dat het mooi is om een aparte `BeamerOutput.vue` en `LivestreamOutput.vue` te gaan maken. Aangezien er nu langzamerhand steeds meer verschil tussen die 2 komt (messages, achtergrond, alpha channel). Maar dit laatste zal ik noteren als nieuw issue.

Reply:
===  verwerkt, 

2e deel begrijp ik nog niet; maar gaat over de algemene output denk ik?

Reply:
Ja klopt, dat bedoel ik inderdaad. Is meer een refactor voor later